### DISCUSSION TITLE
Unwanted profile loads when I press the OFF button between roasts

### DISCUSSION THREAD

**Discussion by embercoffeeco** [artisan-roaster-scope/artisan#1105] [Q&A] [answered]
At the end of a roast, I press **OFF** to reset and prepare for the next batch. When I do that, a different profile loads in the background. It throws me off because the roast information is still plotted on the screen but it's (now) compared against the wrong profile/background. What once looked like a pretty on-target roast now looks off.

For the life of me, I can't figure out what's going on. It feels to me like when I press **OFF** that it loads some default background. Either that's tied to the **OFF** button or some general setting. Either way, I'm looking for **suggestions on how to maintain the loaded profile as the background image when I press the OFF button.**

Video below to show what's occurring:
https://drive.google.com/file/d/17iTXf3QJfOlxiWqvN97AUaeAWS7rU70J/view?usp=sharing

**Comment by poundy** [comment]
You'll need to save out your settings file and attach them here.  To do this export them using menu Help >> Save Settings into a file named artisan-settings.aset. Rename that file to end with .txt and then drag-and-drop them to a message box here on this discussion thread on the Github Web page. 
We have seen a behaviour similar to this where the settings file was read-only and that meant the completed roast wasn't able to adjust the incremental settings. Perhaps you can check to see if MAcOS thinks that file is read-only.
Otherwise, there's some setting in your settings that is causing this behaviour - that's why we need your settings

**Reply by embercoffeeco** [reply]
Oh, of course. That makes sense. Settings attached.
[artisan-settings.txt](https://github.com/artisan-roaster-scope/artisan/files/10784910/artisan-settings.txt)

**Comment by roasterdave** [comment, marked as answer]
This is a settings issue.  You have a button defined, button 13, that loads _435in11m30s.alog_ as the background profile.  This button action is linked to the OFF button and also to the RESET, ON and DROP buttons.  Thus every time OFF, RESET, ON or DROP are pressed that background will be loaded.  Just go to Config>> Events>> Config tab and remove that action from the OFF button (and any of the others if necessary).

**Reply by embercoffeeco** [reply]
Thank you so much. I really appreciate this!
